packages/dds/merge-tree/src/test/client.localReference.spec.ts

```diff
@@ -629,81 +630,112 @@ describe("MergeTree.Client", () => {
 		assert.equal(client2.localReferencePositionToPosition(localRef), 0);
 	});
 
-	it("doesn't crash for remove ref then link to undefined", () => {
-		const client1 = new TestClient();
-		const client2 = new TestClient();
-
-		client1.startOrUpdateCollaboration("1");
-		client2.startOrUpdateCollaboration("2");
-
-		let seq = 0;
-		const insert1 = client1.makeOpMessage(client1.insertTextLocal(0, "abcdef"), ++seq);
-		client1.applyMsg(insert1);
-		client2.applyMsg(insert1);
-
-		const segInfo = client1.getContainingSegment(3);
-
-		assert(segInfo.segment);
-
-		const localRef = client1.createLocalReferencePosition(
-			segInfo.segment,
-			segInfo.offset,
-			ReferenceType.SlideOnRemove,
-			undefined,
-		);
-
-		assert.equal(localRef.getSegment(), segInfo.segment);
-
-		assert(segInfo.segment.localRefs);
-		assert(!segInfo.segment.localRefs.empty);
-
-		segInfo.segment.localRefs.removeLocalRef(localRef);
-		assert(segInfo.segment.localRefs.empty);
-		(localRef as any).link(undefined, 0, undefined);
-		assert(segInfo.segment.localRefs.empty);
-
-		assert.equal(segInfo.segment.localRefs.empty, true);
-		assert.equal(segInfo.segment.localRefs.has(localRef), false);
-		assert.equal(localRef.getSegment(), undefined);
-		assert.equal(localRef.getOffset(), 0);
+	const tgCases = [
```

Comment:
Does this repro the crash, or just adding more coverage?

Reply:
it reproduces the issue reported for the `TrackingGroup` case. The other cases passed before (but still good to check)